### PR TITLE
Add Linux VM spectrum viewer problem to known bugs

### PR DESCRIPTION
**Pull request by rosteen**
Does what it says on the tin. This is the workaround for https://github.com/glue-viz/bqplot-image-gl/issues/94.

### CODE REVIEW

**Review by duytnguyendtn — APPROVED** [review]
Doc build renders and the entry succinctly explains the problem and work around. Thanks!

File: docs/known_bugs.rst [hunk 1]
```diff
@@ -84,6 +84,17 @@ where you do not see this error or use Jupyter Notebook instead.
 Cubeviz
 -------
 
+Spectrum does not appear when running on a VM
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (pllim) [inline docs/known_bugs.rst:88]:
```suggestion
Spectrum does not appear when running on a Linux VM
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

File: docs/known_bugs.rst [hunk 2]
```diff
@@ -84,6 +84,17 @@ where you do not see this error or use Jupyter Notebook instead.
...
+
+When running Jdaviz on a virtual Linux machine, the spectrum may not appear
```
Comment (pllim) [inline docs/known_bugs.rst:90]:
```suggestion
When running Jdaviz on a Linux virtual machine (VM), the spectrum may not appear
```

File: docs/known_bugs.rst [hunk 3]
```diff
@@ -84,6 +84,17 @@ where you do not see this error or use Jupyter Notebook instead.
...
+in the spectrum viewer. This is a known bug in an underlying package. Until
```
Comment (pllim) [inline docs/known_bugs.rst:91]: Can we link to that issue upstream?

File: docs/known_bugs.rst [hunk 4]
```diff
@@ -126,6 +137,11 @@ calculate them on dithered images linked by WCS *unless* you are on the referenc
 Specviz
 -------
 
+Spectrum does not appear when running on a VM
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (pllim) [inline docs/known_bugs.rst:141]:
```suggestion
Spectrum does not appear when running on a Linux VM
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

File: docs/known_bugs.rst (hunk 3 above)
Comment (rosteen) [reply docs/known_bugs.rst:91]: We could, I just didn't think the average user would care.

File: docs/known_bugs.rst [hunk 5]
```diff
@@ -84,6 +84,17 @@ where you do not see this error or use Jupyter Notebook instead.
 Cubeviz
 -------
 
+Spectrum does not appear when running on a Linux VM
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When running Jdaviz on a Linux virtual machine (VM), the spectrum may not appear
+in the spectrum viewer. This is a `known bug <known-gl-issue>` in an underlying
```
Comment (pllim) [inline docs/known_bugs.rst:91]: Where is `known-gl-issue` defined? I don't see it in the diff here.

Comment (rosteen) [reply docs/known_bugs.rst:91]: Oh ffs, I had the wrong text in my clipboard when I pasted

File: docs/known_bugs.rst [hunk 6]
```diff
@@ -84,6 +84,18 @@ where you do not see this error or use Jupyter Notebook instead.
 Cubeviz
 -------
 
+Spectrum does not appear when running on a Linux VM
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When running Jdaviz on a Linux virtual machine (VM), the spectrum may not appear
+in the spectrum viewer. This is a 
+`known bug <https://github.com/glue-viz/bqplot-image-gl/issues/94>` in an underlying
```
Comment (pllim) [inline docs/known_bugs.rst:92]:
```suggestion
`known bug <https://github.com/glue-viz/bqplot-image-gl/issues/94>`_ in an underlying
```

**Review by pllim — APPROVED** [review]
Content LGTM though I think you need a trailing underscore for the link to render properly?

**Pull request merged**